nickfong/CS160-Collaboration
# Merge frontend-integration (formerly integration-all) into master

Created a new branch called `frontend-integration` based off of the most recent version of `integration-all`.  This branch is rebased off of `master`, so it includes the code from `bart-service`.  

Rebase was clean after manually resolving conflicts.  

To Verify:
1. `pull --rebase integration-all`
2. Check out `frontend-integration`
3. Navigate to the `bartgo` directory
4. Run `git diff integration-all MainActivity.java` to show the changes between `frontend-integration` and `integration-all` -- that is, the changes that were made during the rebase
5. Compare the diff to `git diff 88d0dc0 ce8555b MainActivity.java` -- that is, the changes that were made on the `bart-service` branch.  

Creating PR as a formality again.  Will self-assign and merge into master, depreciate old branches, etc.
